src/wet-lab/design/solid-phase.md

```diff
@@ -1 +1,51 @@
+
+For controlled single nucleotide insertion, a robust yet readily cleavable protecting group is required. The primary method of 3’ protection for 5’ to 3’ synthesis is photolabile protecting groups. These groups are efficiently cleaved when exposed to UV light. These groups have been developed due to the desire to generate complex microarrays for genetic analysis. With photolabile protecting groups, a desired pattern of unique oligonucleotides may be synthesized. This chemistry can be leveraged for the use of TDT single nucleotide insertion.
+
+## Goals
+The goal of this phase is 
+  1. To functionalize the surface for the immobilization of primers
+  2. To immobilize the primer 5’ end on the solid phase at a high enough density and selectivity to enable efficient DNA synthesis
+  3. To characterize the density of primer immobilization
+  4. Lastly, the primer and newly synthesized DNA strand must be cleaved from the solid phase
+
+## The actual design of immobilization
+
+There are two different methods being considered for immobilization. All approaches are based on the same principle. 
+
+Disulfide 5’ modifier and dibenzocylooctyne 5’ modifier are the primer methods in consideration. The idea with each modified primer is to use orthogonal reacting functional groups. As a DNA oligo has both a 3’ and 5’ free hydroxyl group, using a 5’ modifier that selectively reacts with its counterpart would prevent the 3’ hydroxyl from reacting in any meaningful quantities.
+![Sulfide solid phase synthesis](Sulfide-SPS.png)
+Rogers, Y.-H.; Jiang-Baucom, P.; Huang, Z.-J.; Bogdanov, V.; Anderson, S.; Boyce-Jacino, M. T. Immobilization of Oligonucleotides onto a Glass Support via Disulfide Bonds: A Method for Preparation of DNA Microarrays. Analytical Biochemistry 1999, 266 (1), 23–30. DOI:10.1006/abio.1998.2857.
+
+## The actual design of modified DNTPs
+
+The DNTPs will be ordered as 3’-BzNPPOC modified DNTP’s or 3’aminohydroxy modified DNTPs. BzNPPOC is a highly photolabile group that is readily cleaved in UV light. This protecting group has been highly used in chip chemistry however care is required to prevent premature deprotection when exposed to ambient light. 3’aminohydroxyl groups are readily cleaved in sodium nitrite. These modified nucleotides will allow for single insertion as there is no 3’ hydroxyl for additional reaction. Thus a specific synthesis can be achieved. As part of the tdt synthesis workflow, an additional deprotection step is required prior to the following nucleotide reaction.
+
+![Photolabile-PG](Photolabile-PG.png)
```

Comment:
For images, add a new line before and after the image so the image is on it's own line and there isn't text beside the image.

Refer to previous comment about images.